ckanext/xloader/tests/test_loader.py

```diff
@@ -1049,7 +1049,7 @@ def test_boston_311(self, Session):
                 u"",
                 u"ONTIME",
                 u"Open",
-                u" ",
+                u"",  # " " transforms to ""
```

Comment:
Why does it transform to empty string here but `None` in other tests?

Reply:
@ThrawnCA I believe that is in the `TypeConverter` code. None values are given an empty string and text type. So I think this just matches what guess types does already.
```
if cell_value is None:
    row[cell_index] = ''
```

Should there be parody between the load_csv and load_table for the `None` values? I don't think that would affect loading into the Datastore at all. As long as the `TypeConverter` can handle `None` values, should be fine right?

cc: @duttonw

Reply:
@ThrawnCA @duttonw okay I think I got string type cells that are blank to get `None` value now.